definitions/benefit/sheets/CaseField/CaseField.json

```diff
@@ -532,5 +532,9 @@
   {"LiveFrom": "07/08/2024", "CaseTypeID": "Benefit", "ID": "shouldReadyToListBeTriggered", "Label": " ", "FieldType": "YesOrNo", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "07/08/2024", "CaseTypeID": "Benefit", "ID": "selectNextHmcHearingType", "Label": "Do you want to select next hearing type i.e. Substantive or Directions hearing", "FieldType": "YesOrNo", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "10/09/2019", "CaseTypeID": "Benefit", "ID": "hmcHearingType", "Label": "Next hearing type", "FieldType": "FixedRadioList", "FieldTypeParameter": "FL_hmcHearingType", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "18/10/2024", "CaseTypeID": "Benefit", "ID": "completedHearingsList", "Label": "Completed Hearings", "FieldType": "Collection", "FieldTypeParameter": "hearingDetails", "SecurityClassification": "PUBLIC"}
+  {"LiveFrom": "18/10/2024", "CaseTypeID": "Benefit", "ID": "completedHearingsList", "Label": "Completed Hearings", "FieldType": "Collection", "FieldTypeParameter": "hearingDetails", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "13/03/2025", "CaseTypeID": "Benefit", "ID": "ftaRequestType", "Label": "New Request", "FieldType": "FixedRadioList", "FieldTypeParameter": "FL_ftaRequestType", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "13/03/2025", "CaseTypeID": "Benefit", "ID": "ftaRequestTopic", "Label": "Request Topic", "FieldType": "FixedList", "FieldTypeParameter": "FL_ftaRequestTopic", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "13/03/2023", "CaseTypeID": "Benefit", "ID": "ftaRequestQuestion", "Label": "Request Question", "FieldType": "TextArea", "Max": 500, "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "13/03/2023", "CaseTypeID": "Benefit", "ID": "ftaCommunications", "Label": "Request Question", "FieldType": "Collection", "FieldTypeParameter": "ftaRequest", "SecurityClassification": "PUBLIC"}
 ]
```

Comment:
For requirement 5/6, add new CaseField entries as follows:
{"LiveFrom": "13/03/2025", "CaseTypeID": "Benefit", "ID": "ftaCommunicationFilter", "Label": "FTA Communication", "FieldType": "FixedList", "FieldTypeParameter": "FL_ftaCommunicationFilter", "SecurityClassification": "PUBLIC"},
{"LiveFrom": "13/03/2025", "CaseTypeID": "Benefit", "ID": "tribunalCommunicationFilter", "Label": "Tribunal Communication", "FieldType": "FixedList", "FieldTypeParameter": "FL_tribunalCommunicationFilter", "SecurityClassification": "PUBLIC"}

Reply:
Done